### ISSUE TITLE
Question about Training Settings

### ISSUE THREAD

**Issue by LittleQuteSweetie** [IIPCVLAB/LCCNet#3]
Hello, thanks for sharing the code !
We are highly interested in this great work and planing to run it on our custom dataset, and we want to have a further understanding about the training settings. It is said in the paper that 
> we train our proposed calibration network on two Nvidia GP100 GPU with batch size 120 and total epochs 120 

, and it seems in the train_with_sacred.py that the number of GPU and the batch size are both doubled, yet the initial learning rate is not, could this be a problem in re-implementation? And we find it would take about 100 hours to train the model from scratch on two Tesla V100 GPU, and would like to know how long does it take when trained on your 4 GPUs, thanks!

**Comment by ghost** [comment]
Same question here, is it possible to share the training log? Thanks!

**Comment by liyang159357** [comment]
I feel the paper was faked

**Comment by ccfendouing** [comment]
I also have the same question.Can you tell me the lr and the batchsize and gpu num?

**Comment by chenpengxin** [comment]
I tried with batch_size=240, max_epoch=120(120 for the first and 50 for others) using two quadro gv100.   It tooks 1 hour per epoch.  Remember there are 5 weights to be trained.  So it will take 120 + 4*50 =320 hours = 13 days to train the model!

**Comment by StiphyJay** [comment]
> I tried with batch_size=240, max_epoch=120(120 for the first and 50 for others) using two quadro gv100. It tooks 1 hour per epoch. Remember there are 5 weights to be trained. So it will take 120 + 4*50 =320 hours = 13 days to train the model!

Another problem. In your pretrained model, the epoch in every model is 300. So do you really train the model with 120epoch to get the paper reported result?

**Comment by liyang159357** [comment]
您好！您的来信我已经收到，我会尽快给您回复。谢谢！——李杨
